16_permutations/permutations.spec.js

```diff
@@ -0,0 +1,64 @@
+const permutations = require("./permutations");
+
+describe("permutations", () => {
+  test("1 possible permutation for a set containing 0 numbers", () => {
+    expect(permutations([]).sort()).toEqual([[]].sort());
+  });
+
+  test.skip("1 possible permutation for a set containing 1 number", () => {
+    expect(permutations([1]).sort()).toEqual([[1]].sort());
+  });
```

Comment:
Nit and only a suggestion - I feel like omitting the `.sort()` with these makes it easier to read, and of course no change in behaviour. We only need the `sort()` for the other ones that have more than one element in the returned array.

```suggestion
  test("1 possible permutation for a set containing 0 numbers", () => {
    expect(permutations([])).toEqual([[]]);
  });

  test.skip("1 possible permutation for a set containing 1 number", () => {
    expect(permutations([1])).toEqual([[1]]);
  });
```

Reply:
yea good idea!